### ISSUE TITLE
CC2652RB Error: Coordinator failed to start, probably the panID is already in use after restart

### ISSUE THREAD

**Issue by robertodocampo** [Koenkk/zigbee2mqtt#6302]
Previus to migrate from my old CC2531 to a new CC2652 i create a laboratory test with only 1 coordinator (CC2652) and 1 device (Blitwoiif BW-SHP13). 

When i start zigbee2mqtt no problems, i pair the device without problems and the system seems to be ok. My problems occurs after restart zigbee2mqtt. After the restart the zigbee2mqtt fails with "Error: Coordinator failed to start, probably the panID is already in use"

I try to delete the config, change several panId's network keys.. and the problems is the same. panId in use after reboot.
I simulate the same scenary with the CC2531 and no problems after Reboot.

I'm ussing zigbee2mqtt@1.17.1  and the latest firmware.

Thanks a lot

**Comment by Koenkk** [comment]
Are you flashing the sick with windows?

**Comment by robertodocampo** [comment]
No, i'm using mac os x.

I reproduce the logs and are available on this url's:

Initial start (and pairing):
https://pastebin.com/bpQEsU6E


After restart the zigbee2mqtt:
https://pastebin.com/VS0jhj7n

Regards!

**Comment by Koenkk** [comment]
Can you try flashing the stick on a different computer? I've spend lots of time investigating the exactly the same issue in https://github.com/Koenkk/zigbee-herdsman/issues/282 . Turned out somehow his setup flashed the firmware incorrectly.

**Comment by robertodocampo** [comment]
Great!! I try to reflash the CC2652 on a W10 (instead of OS X) and the CC2652 survive to the restart!

Thanks a lot @Koenkk !

**Comment by Koenkk** [comment]
What python version is installed on windows/osx?

**Comment by robertodocampo** [comment]
Sorry i deleted the OS X installation but it's possible that i used an old version 3.6 (or similar). On windows i use 3.9.1

regards!

**Comment by benek984** [comment]
@Koenkk I have the same problem, I have tried everything. Re-flashing on MAC and Windows, tried to wrap stick in the aluminium foil (film) still the same. When I start with new PANID I am able to start z2m but soon as I add one device and restart the z2m I am not able to run it. I have CC2652RB from SLAE.SH 
Before I used Texas Instruments CC2531 and all was perfect until my network grow above 30 devices so I decided to go for CC2652RB from SLAE.SH but since then nothing but trouble. Running z2m as add-on in Home assistant. 

Please help

**Comment by kotjaraxx** [comment]
Тоже имею такую проблему. Перепрошивка на разных пк, замена стиков(3шт), проблему не решает.

**Comment by heapxor** [comment]
What is the solution for that?

**Comment by ovizii** [comment]
Another user with CC2652RB from SLAE.SH here. Flashed like this: https://slae.sh/projects/cc2652/#flashing 
I tried flashing from Win 10, later on via Debian. Flashing was successful, I can see the correct firmware on the stick but zigbee2mqtt kees failing with:

```
Zigbee2MQTT:error 2022-06-13 19:19:36: Exiting...
Zigbee2MQTT:error 2022-06-13 19:19:37: Error: network commissioning timed out - most likely network with the same panId or extendedPanId already exists nearby
```
I definitely do not have the same panID. I am trying to set up this new stick on a raspi for a holiday home. My main HA with Z2M is running on my NAS and has a different panID.

Anyway, I'll try and debug this with the instructions provided here https://slae.sh/projects/cc2652/#flashing I just thought I'd post here too, maybe someone has meanwhile figured out what the issue was.

**Comment by jerrychong25** [comment]
> Another user with CC2652RB from SLAE.SH here. Flashed like this: https://slae.sh/projects/cc2652/#flashing I tried flashing from Win 10, later on via Debian. Flashing was successful, I can see the correct firmware on the stick but zigbee2mqtt kees failing with:
> 
> ```
> Zigbee2MQTT:error 2022-06-13 19:19:36: Exiting...
> Zigbee2MQTT:error 2022-06-13 19:19:37: Error: network commissioning timed out - most likely network with the same panId or extendedPanId already exists nearby
> ```
> 
> I definitely do not have the same panID. I am trying to set up this new stick on a raspi for a holiday home. My main HA with Z2M is running on my NAS and has a different panID.
> 
> Anyway, I'll try and debug this with the instructions provided here https://slae.sh/projects/cc2652/#flashing I just thought I'd post here too, maybe someone has meanwhile figured out what the issue was.

Hi @ovizii ,

I had same issue.

Do you found a solution for this issue?

If yes, mind to share full steps?

Thank you!

**Comment by ovizii** [comment]
> Hi @ovizii ,
> 
> I had same issue.
> 
> Do you found a solution for this issue?
> 
> If yes, mind to share full steps?
> 
> Thank you!

I got it working, but I can't remember what the fix was. I had 3 sticks around, I eventually ended up flashing all 3 of them from a Windows machine, if I remember correctly I used this tool https://www.ti.com/tool/UNIFLASH I found referenced here: https://slae.sh/projects/cc2652/#flashing - it was a pain in the a** to get all of that working. Good luck.
